### PR TITLE
fix: safari image flexing issue in onboarding frontend

### PR DESCRIPTION
**Pull request by surbhi-posthog**
## Problem

Related to issue https://posthoghelp.zendesk.com/agent/tickets/20521. (moved to PostHog: https://us.posthog.com/project/2/support/tickets/22914)

The image was not rendering correctly specifically in Safari. Its seemed to work as expected in Google chrome

## Changes

Before:
(In Safari only)
image

After:
(Verified both safari and chrome look as expected)
image

## Does this work well for both Cloud and self-hosted?
Yes

## How did you test this code?

1. Updated code and verified on localhost on safari and google chrome: http://localhost:8000/project/2/onboarding/product_analytics?step=plans

### CODE REVIEW

**Review by patricio-posthog — APPROVED** [review]
Looks good to me! 🚀

File: frontend/src/scenes/onboarding/OnboardingBillingStep.tsx [hunk 1]
```diff
@@ -65,15 +65,18 @@ export const OnboardingBillingStep = ({
                     {product.subscribed && (
                         <div className="mb-8">
                             <div className="bg-success-highlight rounded p-6 flex justify-between items-center">
-                                <div className="flex gap-x-4">
-                                    <IconCheckCircle className="text-success text-3xl mb-6" />
-                                    <div>
+                                <div className="flex gap-x-4 min-w-0">
+                                    {/* min-w-0 to prevent overflow */}
```
Comment (raquelmsmith) [inline frontend/src/scenes/onboarding/OnboardingBillingStep.tsx:69]: generally you don't need to leave comments for CSS changes, so I'd remove those.

If you do leave comments on logic to explain it, we generally put the comment above the line in question 👍

**Review by raquelmsmith — DISMISSED** [review]

**Comment by raquelmsmith** [comment]
Oops, hit submit too soon, meant to also include "nice! thanks for fixing this!" 😄

File: frontend/src/scenes/onboarding/OnboardingBillingStep.tsx (hunk 1 above)
Comment (surbhi-posthog) [reply frontend/src/scenes/onboarding/OnboardingBillingStep.tsx]: removed the comments. Thanks for the feedback

**Review by zlwaterfield — APPROVED** [review]
Awesome, thanks for fixing this!

First PR, woo!

**Pull request merged**